### PR TITLE
Bug #53535: add udm_properties to client

### PR DESCRIPTION
**Pull request by twenzel225**
Support the new attribute udm_properties, which was added to Schools and SchoolClasses in ucschool-kelvin-rest-api 1.5.0

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -15,6 +15,8 @@ To install *Python UCS\@school Kelvin REST API Client*, run this command in your
     $ pip install kelvin-rest-api-client
 
 This is the preferred method to install *Python UCS\@school Kelvin REST API Client*, as it will always install the most recent stable release.
+The major version of the *Python UCS\@school Kelvin REST API Client* must be at least as high as the version of the *UCS\@school Kelvin REST API*
```
Comment (dansan) [inline docs/installation.rst]: `The major and the minor version of the....` `The patch level may differ.`

File: ucsschool/kelvin/client/school_class.py [hunk 2]
```diff
@@ -49,11 +49,19 @@ def __init__(
...
         session: Session = None,
     ):
-        super().__init__(name=name, ucsschool_roles=ucsschool_roles, dn=dn, url=url, session=session)
+        super().__init__(
+            name=name,
+            ucsschool_roles=ucsschool_roles,
+            dn=dn,
+            url=url,
+            session=session,
+            udm_properties=udm_properties,
```
Comment (dansan) [inline ucsschool/kelvin/client/school_class.py]: Keep order of argument consistent. `udm_properties` follows `ucsschool_roles` and is before `dn`.

File: ucsschool/kelvin/client/user.py [hunk 3]
```diff
@@ -131,7 +130,14 @@ def __init__(
...
         session: Session = None,
     ):
-        super().__init__(name=name, ucsschool_roles=ucsschool_roles, dn=dn, url=url, session=session)
+        super().__init__(
+            name=name,
+            ucsschool_roles=ucsschool_roles,
+            dn=dn,
+            url=url,
+            session=session,
+            udm_properties=udm_properties,
```
Comment (dansan) [inline ucsschool/kelvin/client/user.py]: Keep order of argument consistent. `udm_properties` follows `ucsschool_roles` and is before `dn`.

**Comment by SamuelYaron** [comment]
Merge was done on internal mirror -- closing PR

**Pull request closed without merging**